code/modules/mining/boulder_processing/boulder.dm

```diff
@@ -0,0 +1,279 @@
+#define GOLEM_BREAK_SPEED_MULTIPLIER 3
+
+/**
+ * The objects that ore vents produce, which is refined into minerals.
+ */
+/obj/item/boulder
+	name = "boulder"
+	desc = "This rocks."
+	icon_state = "ore"
+	icon = 'icons/obj/ore.dmi'
+	item_flags = NO_MAT_REDEMPTION
+	throw_range = 2
+	throw_speed = 0.5
+	slowdown = 1.5 // It's a big rock.
+	drag_slowdown = 1.5 // It's still a big rock.
+	///When a refinery machine is working on this boulder, we'll set this. Re reset when the process is finished, but the boulder may still be refined/operated on further.
+	var/obj/machinery/bouldertech/processed_by = null
```

Comment:
From what i can see this var isn't being used anywhere. Maybe remove it all together?

Reply:
This one is used in a few places, primarily as a sanity check for the BRM.